modules/electromagnetics/test/tests/auxkernels/azimuthal_Faradays_law/tests

```diff
@@ -7,6 +7,20 @@
     design = 'AzimuthMagneticTimeDerivScalar.md'
     issues = '#28758'
   []
+  [scalar_azim_magnetic_time_deriv_csv]
+    type = CSVDiff
+    command = 'scalar_azim_magnetic_time_deriv_convergence.py'
+    csvdiff = 'scalar_azim_magnetic_time_deriv_convergence.csv'
+    requirement = 'The system shall include a tool for writing spatial convergence data for the time derivative of the azimuthal component of the magnetic field based on a supplied electric field scalar components.'
+    design = 'AzimuthMagneticTimeDerivScalar.md'
+    issues = '#28758'
+    heavy = True
+    recover = false
+    valgrind = none
+    method = opt
+    required_python_packages = 'sympy'
```

Comment:
No problem, will add. Quick question:
Should MOOSE related python packages be include in `required_python_packages =` (like `mooseutils`, `moosetree`, `mms`)? I am assuming no, because these packages are included in the MOOSE python directory, but I wanted to double check.

Reply:
the moose ones are always included so you dont need them in the list